### PR TITLE
Search sort

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
-added search feature
-added sortby feature

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]
Code looks clean, great work

File: src/components/search/Search.jsx [hunk 1]
```diff
@@ -22,7 +32,7 @@ function Search() {
                 ></path>
               </svg>
             </div>
-            <input type="text" class="input input-style input-search" />
+            <input type="text" class="input input-style input-search" value={filterState.search} onChange={changeHandler}/>
```
Comment (Pratik1005) [inline src/components/search/Search.jsx]: When we type in search the first letter does not print, I think value should be search to solve this.

**Review by Pratik1005 — APPROVED** [review]
Clean UI

File: src/components/search/Search.jsx (hunk 1 above)
Comment (mohammadkhizerkhan) [reply src/components/search/Search.jsx]: fixed

**Pull request merged**